app/mailers/response_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class ResponseMailer < ActionMailer::Base
+  domain = ENV.fetch('MAILGUN_DOMAIN', 'roqua.nl')
+  default from: "noreply@#{domain}"
+
+  def response_email(requester_email:, response_uuid:)
+    @link = "future_link_to_finished_response/#{response_uuid}"
+
+    mail to: requester_email,
+         subject: "Resultaten screening #{response_uuid}"
```

Comment:
Ja het is tijdelijk, maar het lijkt me wel wenslijk om de e-mail gelijk zo presenteerbaar mogelijk te maken in deze PR, vandaar mijn comment.